File: Assignment__22/Question3.java
import java.util.Scanner;

class ArrayX
{
    public int Arr[];
    public int iSize;

    public ArrayX(int iNo)
    {
        iSize = iNo;

        Arr = new int [iSize];    //Need to remember
    }

    public void Accept()
    {
        Scanner sobj1 = new Scanner(System.in);

        int iCnt = 0;

        System.out.println("Enter the numbers : ");

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            Arr[iCnt] = sobj1.nextInt();
        }
    }

    public void Display()
    {
        System.out.println("The elements of the array are : ");

        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            System.out.println(Arr[iCnt]);
        }
    }

    public boolean CheckEleven()
    {
        int iCnt = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] == 11)
            {
                return true;
            }   
        }
        return false;

    }
}

class Question3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the number of elements : ");
        int iValue = sobj.nextInt();

        boolean bResult = false;

        ArrayX aobj = new ArrayX(iValue);
        aobj.Accept();
        aobj.Display();

        bResult = aobj.CheckEleven();

        if(bResult == true)
        {
            System.out.println("The number is present in the given array.");
        }
        else
        {
            System.out.println("The number is not present in the given array.");
        }

        aobj = null;
        sobj = null;

        System.gc();
    }
}

File: Assignment__22/Question3.c
//Accept N numbers from user and return difference between frequency of even number and odd numbers.

#include<stdio.h>
#include<stdbool.h>
#include<stdlib.h>


bool CheckEleven(int Arr[], int iSize)
{
    int iCnt = 0;

    

    for(iCnt = 0; iCnt < iSize; iCnt++)    
    {
        if(Arr[iCnt] == 11)
        {
            return true;
        }
        else
        {
            return false;
        }
    }

}

int main()
{
    int iLength = 0;
    int iCnt = 0;
    int *ptr = NULL;
    bool bResult = false;

    printf("Enter the number of elements : ");
    scanf("%d", &iLength);

    ptr = (int *)malloc(iLength * sizeof(int));
    
    if(ptr == NULL)      
     {
        printf("Unable to allocate the memoey\n");
        return -1;
    }

    printf("Enter the elements : \n");

    for(iCnt = 0; iCnt < iLength; iCnt++)
    {
        scanf("%d",&ptr[iCnt]);
    }

    bResult = CheckEleven(ptr,iLength);

    if(bResult == true)
    {
        printf("The given array contains number 11.");
    }
    else
    {
        printf("The given array does not contain number 11.");
    }

    free(ptr);

    return 0;

}
